source/index.html.md

```diff
@@ -1955,7 +1955,9 @@ Field Name | Type | Description | Units
 }
 ```
 
-Returns a list of details for 1 or more orders
+Returns details for 1 or more of your orders. Only opened or partially filled orders will be returned by this endpoint. 
```

Comment:
Let's fix this to:

...for one or more...

Reply:
Addressed too